openedx/core/djangoapps/content/learning_sequences/api/permissions.py

```diff
@@ -31,20 +27,5 @@ def can_see_all_content(requesting_user: types.User, course_key: CourseKey) -> b
 
     There's no need to run processors to restrict results for these users.
     """
-    return (
-        GlobalStaff().has_user(requesting_user) or
-        CourseStaffRole(course_key).has_user(requesting_user) or
-        CourseInstructorRole(course_key).has_user(requesting_user)
-    )
```

Comment:
These lines were replaced with a call to `has_access('staff')`. Which checks those same roles, plus `OrgStaffRole` and `OrgInstructorRole`. And checks if there's any masquerading happening.

I figured the semantic label of `can_see_all_content` was worth keeping this wrapper method instead of a direct call to `has_access` in the api method.